api-schema/src/main/java/org/stellar/anchor/api/callback/CustomerIntegration.java

```diff
@@ -17,7 +17,7 @@ public interface CustomerIntegration {
    * @return The GET customer response.
    * @throws AnchorException if error happens
    */
-  GetCustomerResponse getCustomer(GetCustomerRequest request) throws AnchorException;
+  CustomerResponse getCustomer(GetCustomerRequest request) throws AnchorException;
```

Comment:
Although `PutCustomerResponse` and `GetCustomerResponse` have the same payload, combining the two classes seems confusing when reading the code. For example, the `Sep12GetCustomerRequest` and `Sep12PutCustomerRequest` gives the code better readability.

Is there better solution not to duplicate the code but keep the readability?

Reply:
I recreated the `PutCustomerReponse` class as a subclass of `GetCustomerResponse`.